iravelmakina/eigenvectors-ivelmakina
# Part three

### Description:
- Added encryption and decryption functions using a key matrix.
- Included examples of encryption and decryption with a sample message and a randomly generated key matrix.

### Changes:
- Added `encrypt_message` function that encrypts a given message using a key matrix.
- Implemented `decrypt_message` function that decrypts an encrypted message using the same key matrix.
- Added sample code demonstrating the usage of these functions with a randomly generated key matrix.

### How to Test:
1. Clone the repository and navigate to the project directory.
3. Run the provided code to generate a key matrix and encrypt the sample message.
4. Verify the encrypted vector output.
5. Run the decryption function using the encrypted vector and the same key matrix.
6. Confirm that the decrypted message matches the original message.

### Screenshots:
4-6. 
![image](https://github.com/iravelmakina/eigenvectors-ivelmakina/assets/150923968/4a84e651-87ca-4c20-bd85-1f5819bd6dbf)
